source/features/copy-file.tsx

```diff
@@ -6,11 +6,11 @@ import {groupSiblings} from '../libs/group-buttons';
 
 function init() {
 	// This selector skips binaries + markdowns with code
-	for (const code of select.all('.file .blob-wrapper > .highlight:not(.rgh-copy-file)')) {
+	for (const code of select.all<HTMLInputElement>('.file .blob-wrapper > .highlight:not(.rgh-copy-file)')) {
 		code.classList.add('rgh-copy-file');
 		const file = code.closest('.file');
 
-		const content = select.all('.blob-code-inner', file)
+		const content = select.all<HTMLInputElement>('.blob-code-inner', file)
```

Comment:
same as comment https://github.com/sindresorhus/refined-github/pull/1750/commits/df7d4f0e8097bf31f89b70d01fd9894e263b7c5f#r254094372

Reply:
No, I'd stick to `HTMLElement`